### PR TITLE
Complete : Weather App Project

### PR DESCRIPTION
**Pull request by imahnama**
## In this project I :
- Set up a blank HTML document
- install webpack
- Write a function that hit the API
- Write functions that process the JSON data am getting from the API and return an object with only the data required
- Set up a simple search input that will let users input their location and will fetch the weather info 
- Style the UI
- Add readme
-Add linter files

### CODE REVIEW

File: src/weather.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable import/no-cycle */
+/* eslint-disable camelcase */
+/* eslint-disable no-useless-concat */
+/* eslint-disable no-unused-vars */
```
Comment (mcrd25) [inline src/weather.js:5]: You shouldn't have to disable this, perhaps you need to improve your code architecture

**Review by mcrd25 — APPROVED** [review]
# Changes Approved :heavy_check_mark: 

Hi @imahnama :wave: 

App  works well. I see nothing that keeps me from passing you. However, consider the following:

- App has three temp rows. Why is this? Consider sticking to one

Good job though! :+1: 

***If you have any questions, now or in the future, do not hesitate to contact me on social media, email or on slack and I will be glad to chat with you.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

File: src/weather.js (hunk 1 above)
Comment (imahnama) [reply src/weather.js:5]: noted

**Comment by imahnama** [comment]
> # Changes Approved ✔️
> Hi @imahnama 👋
> 
> App works well. I see nothing that keeps me from passing you. However, consider the following:
> 
> * App has three temp rows. Why is this? Consider sticking to one
> 
> Good job though! 👍
> 
> _**If you have any questions, now or in the future, do not hesitate to contact me on social media, email or on slack and I will be glad to chat with you.**_
> 
> _Happy coding 😄,_
> **[--Maya](https://github.com/mcrd25)**
> [GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

Hi, @mcrd25 Thanks for approving this PR. I have three rows as the other two are maximum temperature and minimum temperature. I do agree though maybe other data would be more useful e.g wind pressure humidity e.t.c

**Pull request merged**